pluribus/games/short_deck/state.py

```diff
@@ -163,14 +175,6 @@ def apply_action(self, action_str: Optional[str]) -> ShortDeckPokerState:
                     # Distribute winnings.
                     new_state._poker_engine.compute_winners()
                 break
-            else:
-                # The current player isn't active, and we are not terminal.
-                # We'll move to the next player in the next iteration of this
-                # while loop, but append a null action to the history to
-                # signify the notation h · 0 in algorithm 1 of the
-                # supplementary material of the Pluribus paper.
-                new_state._history.append("skip")
```

Comment:
I'm a little skeptical that the history doesn't need to have `"skip"` in it, or some other value that delineates that the player at her turn could not make a move due to not being in the hand.

I'm worried that this will lead to ambiguous infosets.

Reply:
Well I suppose the more I think about it, the action sequence must lead up to some discrete set, and this should be deterministic, so maybe on reflection it doesn't matter whether there is a `"skip"` in the history.

Reply:
Just have a quick consideration on this! Wouldn't want there to be action sequences leading to different infosets

Reply:
I see your point. 

No, given that the player that is skipped belongs to a distinct position after a fold relative to the traversing player (assuming a terminal/show down node has not been reached), adding the skip is actually redundant information. 

This can be seen in my skip test, the logic is such that every n_players after a fold, in a given round, should have a skip. This is distinct to having seen a fold for the player in that position.

That said, I do like the skip from a debugging/modeling the game heuristically POV.

The action sequences without the skip is distinct as shown by the actions_sequences.py script. So I think we're good here.